client/src/pages/EditProfile.js

```diff
@@ -11,6 +11,7 @@ import Chip from "@material-ui/core/Chip";
 
 import { UserContext } from "../contexts/user/UserContextProvider";
 import ProfilePicLoader from "../components/ProfilePicLoader";
+import allCuisines from "../lib/allCuisines";
```

Comment:
I see `MenuItem` and `allCuisines` being added to the import but not being added elsewhere in the file. Do we need these?